src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -971,7 +1002,7 @@ private LLVMValueRef GetEETypeForTypeDesc(TypeDesc target, bool constructed)
         private LLVMValueRef GetEETypePointerForTypeDesc(TypeDesc target, bool constructed)
         {
             ISymbolNode node;
-            if (constructed)
+            if (constructed && !target.IsByRefLike)
```

Comment:
Why is this needed? `IsByRefLike` should be already handled in `MaximallyConstructableType`.

Reply:
Good catch. The change to use ```MaximallyConstructableType``` came in after I sent out the PR, so I just merged it in.